docs/source/architecture/sub_stakes.rst

```diff
@@ -0,0 +1,278 @@
+
+**Example:**
+
+    A staker prolongs sub-stake for 2 additional periods up to 7:
+		- Before: 
+			* 1st sub-stake = 900 tokens for 5 periods
+		- After: 
+			* 1st sub-stake = 900 tokens for 7 periods
+
+    .. aafig::
+        :proportional:
+        :textual:
+
+                         Before             
+
+            stake
+            ^
+            |
+         900+-----------------------+
+            |          1st          |   period
+            +---+---+---+---+---+---+---->
+            | 0 | 1 | 2 | 3 | 4 | 5 | 6      
+			
+			
+			
+                         After             
+
+            stake
+            ^
+            |
+         900+-------------------------------+
+            |              1st              |   period
+            +---+---+---+---+---+---+---+---+---->
+            | 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8
+			
+			
+Sub-stake division
+^^^^^^^^^^^^^^^^^^
+*Methods* : ``divideStake(uint256,uint256,uint16)``
+
+If necessary, stakers can extend the locking duration for only a portion of their tokens in a sub-stake by using the ``divideStake()`` method. This method splits a sub-stake into two elements: the old sub-stake with the reduced locked amount and the new sub-stake with the specified amount. The new sub-stake has the specified locked amount and an extended lock duration, based on the specified number of periods, with the same start period as the old sub-stake.
+
+**Example:**
+
+    A staker divides sub-stake and extends locking time for 300 tokens for 2 additional periods:
+		- Before: 
+			* 1st sub-stake = 900 tokens for 5 periods
+		- After: 
+			* 1st sub-stake = 600 tokens for 5 periods
+			* 2nd sub-stake = 300 tokens for 7 periods
+
+    .. aafig::
+        :proportional:
+        :textual:
+
+                         Before             
+
+            stake
+            ^
+            |
+         900+-----------------------+
+            |                       |
+            |          1st          |
+            |                       |   period
+            +---+---+---+---+---+---+---->
+            | 0 | 1 | 2 | 3 | 4 | 5 | 6       
+			
+			
+			
+                         After             
+
+            stake
+            ^
+            |
+         900+-----------------------+
+            |                       |
+            |          1st          |
+         300+-----------------------+-------+
+            |              2nd              |   period
+            +---+---+---+---+---+---+---+---+---->
+            | 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8
+   
+
+
+Slashing
+^^^^^^^^
+*See:* `the slashing protocol`_ 
+
+.. _`the slashing protocol`: https://docs.nucypher.com/en/latest/architecture/slashing.html
+
+
+
+
+Flags that affect the sub-stake array
+-------------------------------------
+
+Re-staking
+^^^^^^^^^^
+*Used in methods* : ``confirmActivity()``, ``mint()``
+
+When re-staking is turned off, the number of locked tokens in sub-stakes does not change by itself.
```

Comment:
Since we used enabled later on, we should use disabled here.
```suggestion
When re-staking is disabled, the number of locked tokens in sub-stakes does not change by itself.
```

Reply:
✔️